peachjam/forms.py

```diff
@@ -519,10 +521,35 @@ class PeachjamLoginForm(LoginForm):
     captcha = ReCaptchaField(widget=ReCaptchaV2Invisible)
 
 
-class UserForm(forms.ModelForm):
+class UserProfileForm(forms.Form):
+    first_name = forms.CharField(max_length=255, required=False)
+    last_name = forms.CharField(max_length=255, required=False)
+    preferred_language = forms.ModelChoiceField(
+        queryset=Language.objects.filter(iso_639_1__in=get_languages())
+    )
+
     class Meta:
```

Comment:
If it's no longer a ModelForm, then Meta does nothing and can be dropped completely.

Reply:
done